Eng/CodeReview.md

```diff
@@ -0,0 +1,52 @@
+# Code Review
+All of our engineering work makes it to production by first passing through a
+code review. No line of code is too small or inconsequential to bypass this part
+of our product lifecycle.
+
+## Opening a Pull Request
```

Comment:
I think these are great guidelines! I would suggest consistency in final punctuation   🤓  🤓  🤓